consensus/consensus_tests/utils_test.go

```diff
@@ -127,21 +133,20 @@ func CreateTestConsensusPocketNode(
 	utilityMock := baseUtilityMock(t, testChannel)
 	telemetryMock := baseTelemetryMock(t, testChannel)
 
-	bus, err := shared.CreateBus(persistenceMock, p2pMock, utilityMock, consensusMod, telemetryMock)
+	bus, err := shared.CreateBus(config, genesis, persistenceMock, p2pMock, utilityMock, consensusMod.(modules.ConsensusModule), telemetryMock)
 	require.NoError(t, err)
 
 	pk, err := cryptoPocket.NewPrivateKey(cfg.Base.PrivateKey)
 	require.NoError(t, err)
 
-	pocketNode := &shared.Node{
-		Address: pk.Address(),
-	}
+	pocketNode := shared.NewWithAddress(pk.Address())
```

Comment:
This function name is very non-descript. Please make it clearer what object is being returned from `shared` package

Reply:
Agreed 

https://github.com/pokt-network/pocket/pull/235/files#diff-9f97d711d54398bf6257c266207519c11eb1d254c798d578786d0e94713cb164R151